src/main/java/org/example/Student.java

```diff
@@ -0,0 +1,96 @@
+package org.example;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Year;
+import java.util.Date;
+
+import static org.example.Gender.F;
+import static org.example.Gender.M;
+
+public class Student{
+    String firstName;
+    String lastName;
+    String gender;
+    Year currentYear = Year.now();
```

Comment:
you can save this in the app instead of each student, or just call Year.now where needed